src/test/java/uk/gov/companieshouse/confirmationstatementapi/client/OracleQueryClientTest.java

```diff
@@ -308,36 +310,32 @@ void testGetRegisteredEmailAddress() throws ServiceException {
     }
 
     @Test
-    void testGetRegisteredEmailAddressServiceUnavailable() throws ServiceException {
+    void testGetRegisteredEmailAddressServiceUnavailable() {
         // GIVEN
 
-        ResponseEntity<RegisteredEmailAddressJson> response = ResponseEntity.status(HttpStatus.SERVICE_UNAVAILABLE).body(null);
-
         var url = DUMMY_URL + "/company/" + COMPANY_NUMBER + "/registered-email-address";
 
         // WHEN
 
-        when(restTemplate.getForEntity(url, RegisteredEmailAddressJson.class )).thenReturn(response);
+        when(restTemplate.getForEntity(url, RegisteredEmailAddressJson.class)).thenThrow(RestClientException.class);
 
         // THEN
 
         assertThrows(ServiceException.class, () -> oracleQueryClient.getRegisteredEmailAddress(COMPANY_NUMBER));
     }
 
     @Test
-    void testGetRegisteredEmailAddressNoBody() throws ServiceException {
+    void testGetRegisteredEmailAddressCompanyNotFound() {
```

Comment:
Not sure about the name of the test, are we trying to replicate the situation of the CORPORATE_BODy_DETAILS not being present for a company?

Reply:
renamed to testGetRegisteredEmailAddressEmailAddressNotFound